frontend/src/cards/ui/Sources.test.ts

```diff
@@ -0,0 +1,20 @@
+import { stripCountyFips } from "./Sources";
+
+describe("stripCountyFips", () => {
+  it("Should return string without fips code", () => {
+    const output = ["cdc_restricted_data-by_race_county_processed_time_series"];
+    const input = [
+      "cdc_restricted_data-by_race_county_processed_time_series-48",
+    ];
+    expect(stripCountyFips(input)).toEqual(output);
+  });
+  it("Should return input when county level data isn't being displayed", () => {
+    const output = [
+      "cdc_restricted_data-by_race_county_processed_time_series-with_age_adjust",
+    ];
+    const input = [
+      "cdc_restricted_data-by_race_county_processed_time_series-with_age_adjust",
+    ];
+    expect(stripCountyFips(input)).toEqual(output);
+  });
```

Comment:
can we add another test that sends in an array with more than one id, a mix of needing to be stripped / not needing to be stripped? and then expect the stripped array? I think that test would have failed and exposed the nested mapping issue